tests/Resources/Fixtures/config/full.xml

```diff
@@ -62,6 +62,8 @@
             </authentication>
             <cache cache-pool="my_cache_pool" stream-factory="my_other_stream_factory">
                 <config default-ttl="42" cache-lifetime="2592000" hash-algo="sha1" cache-key-generator="null">
+                    <cache-listeners>my_cache_listener_0</cache-listeners>
+                    <cache-listeners>my_cache_listener_1</cache-listeners>
```

Comment:
when you do the fixXmlConfig, this will be `<cache-listener>` (which is the way how proper xml should look, according to symfony)

Reply:
done.